package.json

```diff
@@ -15,12 +15,12 @@
     "commitmsg": "validate-commit-msg",
     "prettier": "prettier --write \"**/*.{js,css,scss}\"",
     "precommit": "lint-staged",
-    "develop": "start-storybook -p 3000",
+    "storybook": "start-storybook -p 3000",
     "test": "jest"
   },
   "dependencies": {
     "carbon-components": "^8.1.2",
```

Comment:
Should we updated `carbon-components` too in this PR for the new node range?

Reply:
Done!